### ISSUE TITLE
Improve titles with institutional affiliation to make them clearer

### ISSUE THREAD

**Issue by ahankinson** [bodleian/fihrist-mss#11]
TL;DR
To support clearer titles, it would be good to have a clear and succinct institution or affiliation to place in the MS title field.

---
Currently in our 'Classmarks' list we just have the shelfmarks with no institutional affiliation listed:

http://fihrist-qa.bodleian.ox.ac.uk/?f%5Btype%5D%5B%5D=manuscript

It would be good to have some data in the TEI that allowed us to give a nice, short rendering of the institution or organization holding the MS; Something like "Bodleian Library MS 3".

I was thinking of doing this by using the 'Institution' or 'Repository' fields; however, this will break down for the "Arabic Commentaries on the Hippocratic Aphorisms project", since this seems to be the value for institution, repository, and collection.

**Comment by eifionjones** [comment]
Don't worry too much about the Arabic Commentaries records - they are a bit of an anomaly and we'll probably handle them slightly differently once the dust has settled

**Comment by eifionjones** [comment]
After discussion with Yasmin, it would actually be better if the affiliation followed the classmark in this case - e.g. "Bodleian Library MS. Arab. c. 40" should be "MS. Arab. c. 40 Bodleian Library"

**Comment by ahankinson** [comment]
Really? That flies in the face of most conventions in referencing shelfmarks that I'm aware of. It's a bit like at the end of the sentence the verb putting. You don't really know which "MS 1" you're looking at until you reach the end.

At the very least the institution should be set off by parentheses: "MS. Arab. c. 40 (Bodleian Library)"

**Comment by eifionjones** [comment]
It was all Yasmin's idea! But yes, maybe try them in parenthesis. Maybe it's something to do with the ordering of the manuscripts, I'll ask her

**Comment by andrew-morrison** [comment]
This is now on the QA site. Reopen this issue if you want to change it back, or try something else.
